MonoGame.Framework.Content.Pipeline/Graphics/DxtTexture/Bitmap.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;

```

Comment:
@KonajuGames @dellis1972 - What should we do about a copyright header here?

I want to give credit to @vpenades work here and his library, but don't want to complicate the licensing of MG.

Reply:
Maybe including the source files directly is not the best option here. Why not include it as a thirdparty lib. We could get started on #6144 with this. Seems like a great idea to me to use NuGet for managed dependencies. We could optionally have the installer pre-download deps to the global nuget cache so users don't have to worry about an internet connection.